lib/util.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const uv = process.binding('uv');
+const Debug = require('vm').runInDebugContext('Debug');
```

Comment:
Yeah – I *thought* it might be a bit fraught with peril; was surprised to see the chrome dev tools expose that info.